ClientLibraryConsoleAppSample/Program.cs

```diff
@@ -11,15 +11,15 @@ namespace ClientLibraryConsoleAppSample
     class Program
     {
         //============= Config [Edit these with your settings] =====================
-        internal const string vstsCollectionUrl = "https://myaccount.visualstudio.com"; //change to the URL of your VSTS account; NOTE: This must use HTTPS
+        internal const string azDevOrganizationUrl = "https://dev.azure.com/myaccount"; //change to the URL of your Azure DevOps account; NOTE: This must use HTTPS
```

Comment:
how about azureDevOpsOrganizationUrl (I know it's long but it follow's the guidelines of not abbreviating the brand).  Also, let's use organization instead of my account